### DISCUSSION TITLE
nnUnetV2- steps

### DISCUSSION THREAD

**Discussion by carlotita22** [MIC-DKFZ/nnUNet#1472] [General]
Hi everyone,

I have doubts about nnUnetV2: 

1. For libraries: 

I installed for colab pytorch and after nnUnetV2, but the following warning appears:

"WARNING: The following packages were previously imported in this runtime:
  [argparse]
You must restart the runtime in order to use newly installed versions."

Is ok? 

2. For data:

I want to try with the data of Decatlon (heart), I need installed the github?
http://medicaldecathlon.com/

The format of data is be must like:

nnUNet_raw/Dataset002_Heart/
├── dataset.json
├── imagesTr
│   ├── la_003_0000.nii.gz
│   ├── la_004_0000.nii.gz
│   ├── ...
├── imagesTs
│   ├── la_001_0000.nii.gz
│   ├── la_002_0000.nii.gz
│   ├── ...
└── labelsTr
    ├── la_003.nii.gz
    ├── la_004.nii.gz
    ├── ...

3. For Fingerprint: 
What is the code for fingerprint, are there examples? because, I searched for examples but couldn't find

4. For the following, I dont for where start,  exist like a route of files?

**Comment by jamesgoldfarb** [comment]
Everything you need in in the "How to Use"
https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/how_to_use_nnunet.md

If you have errors, search the issues and discussion. Make sure to delete "is:open"
